OpenTelemetry.DotNet.Auto.psm1

```diff
@@ -211,6 +211,14 @@ function Install-OpenTelemetryCore() {
 
         # OpenTelemetry service locator
         [System.Environment]::SetEnvironmentVariable('OTEL_DOTNET_AUTO_INSTALL_DIR', $installDir, [System.EnvironmentVariableTarget]::Machine)
+
+        # Register .NET Framweworks dlls in GAC
+        [System.Reflection.Assembly]::Load("System.EnterpriseServices, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")            
+        $publish = New-Object System.EnterpriseServices.Internal.Publish 
+        $dlls = Get-ChildItem -Path $installDir\netfx\ -Filter *.dll -File
+        foreach ($dll in $dlls) {
+            $publish.GacInstall($dll)
```

Comment:
Q.: what happens if the dll is not strong-named?

Reply:
Right now only 3 DLLs (grpc_csharp_ext.x64.dll, grpc_csharp_ext.x86.dll, netstandard.dll) are not added to GAC and if it fails then we have a warning log in Event Viewer 

PS. Fix https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1884/commits/cf1c304388ccc3c1d19db452d0d1597b00ec72c3